app/logic/events.py

```diff
@@ -65,7 +65,7 @@ def attemptSaveEvent(eventData, attachmentFiles = None):
 
     try:
         events = saveEventToDb(newEventData)
-        if attachmentFiles:
+        if  attachmentFiles != None:
```

Comment:
Did `if attachmentFiles` not work? Seems like it would, and that is preferred over a double negative